docusaurus.config.js

```diff
@@ -351,12 +351,12 @@ const config = {
         specs: [
           {
             id: 'management-api',
-            spec: 'https://api.devcycle.com/swagger.json',
+            spec: 'https://api.devcycle.com/openapi.json',
```

Comment:
how come changing this file?

Reply:
swagger is kinda the out-dated name, openAPI is should be used now. I updated the API route to support both.